tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -837,7 +837,7 @@ class SpirvSwitch : public SpirvBranching {
 
   SpirvInstruction *getSelector() const { return selector; }
   SpirvBasicBlock *getDefaultLabel() const { return defaultLabel; }
-  llvm::ArrayRef<std::pair<llvm::APInt, SpirvBasicBlock *>> getTargets() const {
+  std::vector<std::pair<llvm::APInt, SpirvBasicBlock *>> &getTargets() {
```

Comment:
You can revert all the `std::vector` changes if you change the `llvm::ArrayRef` here by a `llvm::MutableArrayRef`.

Reply:
Good call, thanks!